### PR TITLE
Add additional furtive feature

### PR DESCRIPTION
**Pull request by Parcly-Taxel**
This came up in a live discussion.

### CODE REVIEW

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -290,6 +290,8 @@ The student shares notes and questions with other students with Internet connect
 === Login using email and password `[coming in v2.0]`
 The student can register and login to the application using their email and password.
 
+=== Flash card grouping suggestions `[coming in v2.0]`
+NUStudy will intelligently suggest groupings of flash cards based on their content.
```
Comment (irene-lly) [inline docs/UserGuide.adoc:294]: I think it is a good idea!

File: docs/UserGuide.adoc [hunk 2]
```diff
@@ -290,6 +290,8 @@ The student shares notes and questions with other students with Internet connect
 === Login using email and password `[coming in v2.0]`
 The student can register and login to the application using their email and password.
 
+=== Flash card grouping suggestions `[coming in v2.0]`
```
Comment (Auxinnn) [inline docs/UserGuide.adoc:293]: I thinks it's a good point to add in the UserGuide.

Comment (ShirleyWangxt) [inline docs/UserGuide.adoc:293]: This feature has the potential to differentiate our app more from other revision apps. But the actual implementation would be more complex.

**Review by ShirleyWangxt — COMMENTED** [review]
Good v2.0 extension

File: docs/UserGuide.adoc (hunk 1 above)
Comment (shuiyao-sg) [inline docs/UserGuide.adoc:294]: Would it be better to elaborate a bit on the "intelligently"? Something like "intelligently suggest...by machine learning etc."

Comment (Parcly-Taxel) [reply docs/UserGuide.adoc:294]: Machine learning might not be needed.

**Comment by Parcly-Taxel** [comment]
SUCH GALL!!!

**Pull request merged**